Dockerfile

```diff
@@ -1,8 +1,57 @@
-FROM geonode/django
-MAINTAINER GeoNode development team
+FROM python:2.7.9
```

Comment:
We should maintain it here, so please update the dockerfile from the geonode/django repo